### ISSUE TITLE
Space Suggestion: $\mathbb{R}^\infty$

### ISSUE THREAD

**Issue by GeoffreySangston** [pi-base/data#1024]
## Space Suggestion

$X := \mathbb{R}^\infty$ is the subspace of $\mathbb{R}^\omega$ consisting of eventually zero sequences.

I believe $X$ is homeomorphic to the following spaces.

- The [topological union](https://en.wikipedia.org/wiki/Coherent_topology#Topological_union) $\bigcup_{n = 0}^\infty \mathbb{E}^n$, where $\mathbb{E}^n$ denotes $n$-dimensional Euclidean space. (I've seen topology on a direct limit coinduced by the functions $\mathbb{E}^n \to \lim_{n \to \infty} \\{\mathbb{E}^n\\}$, e.g. on page 5 of Spanier, but not topological union. It seems like a nice name for this, but I haven't found a proper reference yet.)
- The subspace of $\ell^2$ consisting of eventually zero sequences. Note that this space is implicit on line 2 of the second list of spaces in [S30](https://topology.pi-base.org/spaces/S000030), and appears in [Bing's paper referenced there](https://www.ams.org/journals/bull/1968-74-05/S0002-9904-1968-12044-0/). It could be nice to link S30 and this space. I kind of want to ask on MSE if $\ell^2 \backslash X'$ is homeomorphic to $\ell^2$ for any embedding $X'$ of $X$, but I'm not sure when I will since it might be a silly question.

For a homeomorphism of $X$ with the first space in this list, or of the first and second spaces, I believe it's simply a matter of applying the [universal property of a direct limit](https://en.wikipedia.org/wiki/Direct_limit#Direct_limits_in_an_arbitrary_category). 

## Rationale

I first learned of this space from Exercise 7 on page 118 of Munkres, but that is probably not the best reference. I'll look around for better references, but haven't found a great one yet.

## Relationship to other spaces and properties

Each $\mathbb{E}^n := \\{x_i = 0, i > n\\} \subset X$ is a closed subset of $X$ which has empty interior. Since $X = \bigcup_{n = 0}^\infty \mathbb{E}^n$, it follows that $X$ is not [Baire](https://topology.pi-base.org/properties/P000064). This distinguishes $X$ from [S30](https://topology.pi-base.org/spaces/S000030), and for me is the justification to include it. The next challenge is to distinguish $X$ from the weak topology on $\ell^2$ [S21](https://topology.pi-base.org/spaces/S000021), which is because [S21 is not Metrizable](https://topology.pi-base.org/spaces/S000021/properties/P000053) but $X$ is metrizable as it is a [subspace of a metrizable space](https://math.stackexchange.com/questions/1559528/subspace-of-metrizable-space-is-metrizable), since [S30 is metrizable](https://topology.pi-base.org/spaces/S000030/properties/P000055). Then $X$ will fill in the empty search:

[π-Base, Search for `~baire + metrizable + connected`](https://topology.pi-base.org/spaces?q=%7Ebaire+%2B+metrizable+%2B+connected)

**Comment by Moniker1998** [comment]
While this is a known space, it would be nice to include which already existing properties does it satisfy that other spaces don't.

**Comment by GeoffreySangston** [comment]
@Moniker1998 I've included this at the bottom. Is there a something wrong with [π-Base, Search for `~baire + metrizable + connected`](https://topology.pi-base.org/spaces?q=%7Ebaire+%2B+metrizable+%2B+connected)?

**Comment by prabau** [comment]
Isn't this space what is called the $\sigma$-product of countably many copies of $\mathbb R$?  (That's "sigma-product" with a lowecase sigma, as opposed to uppercase sigma, which is a different thing, which would also be interesting to have.  Corson compact related?)

**Comment by GeoffreySangston** [comment]
@prabau $\mathbb{R}^\infty$ is the same as the subspace of finitely supported functions in $\mathbb{R}^\omega$. It fits the pattern of [S000181](https://topology.pi-base.org/spaces/S000181), but I am not familiar with the term '$\sigma$-product', and that page doesn't seem to have a reference with a general definition. What is your reference for $\sigma$-product?

**Comment by prabau** [comment]
Encyclopedia of General Topology was also my source for this.

$\sigma$-product in also mentioned in @ccaruvana 's answer to https://math.stackexchange.com/questions/4736734/a-space-which-is-sigma-compact-but-neither-hemicompact-nor-second-countable, which was the basis for space S181.

I think it would make sense to use $\sigma$-product for the name of the space.  It's a well known concept, and gives a short and unambiguous description, if one knows the terminology.

**Comment by prabau** [comment]
As for the specific notation, see the discussion Steven and I had for #899.  As he mentions, there is no standardized notation here, so we picked one that seemed reasonable.

**Comment by prabau** [comment]
As for Munkres, it's a good book, but somewhat idiosyncratic in its notation and terminology sometimes.  Was meant as an undergraduate text for MIT, not a graduate textbook.  So we may not need to take his notation as face value.

**Comment by GeoffreySangston** [comment]
> Encyclopedia of General Topology was also my source for this.
> 
> σ -product in also mentioned in @ccaruvana 's answer to https://math.stackexchange.com/questions/4736734/a-space-which-is-sigma-compact-but-neither-hemicompact-nor-second-countable, which was the basis for space S181.

It's mentioned, but defined for a particular space. I think it makes sense to give references to general concepts which may not be known outside of general topology circles whenever they're used in special cases. E.g., the situation with density, or 'join', and here. It seems to limit the utility of pi-base significantly to expect to user to be an expert in general topology.

**Comment by prabau** [comment]
I was just mentioning the mathse post as an example using that terminology, but never suggested to include that.  Referencing Encycl. of Gen Top is the right thing to do.

**Comment by Moniker1998** [comment]
> Maybe it's worth clarifying what the level of knowledge expected from the ordinary user of pi-base is?

I don't think any level of knowledge is expected. To verify properties of some examples, necessarily some advanced tools would have to be used, for others, elementary knowledge suffices. 

In this particular example, lots of people in algebraic topology seem to use "$\mathbb{R}^\infty$" as the name for this space. It'd be good to include multiple aliases.

**Comment by GeoffreySangston** [comment]
@prabau I didn't think you did by the way. That was actually a comment about the page [S000181](https://topology.pi-base.org/spaces/S000181)

**Comment by prabau** [comment]
@Moniker1998 

> In this particular example, lots of people in algebraic topology seem to use " R ∞ " as the name for this space.

Can you give an example of use in algebraic topology?

**Comment by prabau** [comment]
> @prabau I didn't think you did by the way. That was actually a comment about the page [S000181](https://topology.pi-base.org/spaces/S000181)

@GeoffreySangston I agree that the page for S181 could mention the general construction of sigma-product with a suitable reference.

**Comment by Moniker1998** [comment]
> Can you give an example of use in algebraic topology?

Pay in mind that this is not my field, but this example is a colimit of $\mathbb{R}^n$ and a CW-complex, that's why its important in algebraic topology.
See [here](https://mathoverflow.net/questions/73246/why-is-mathbbr-infty-defined-the-way-it-is) for one example.

**Comment by GeoffreySangston** [comment]
The superscript $\infty$ notation is common in bundle theory, when discussing classifying spaces. See page 2 of Husemoller's book *Fibre bundles*, where $\mathbb{R}^\infty$, $\mathbb{C}^\infty$, $RP^\infty$, etc... are defined as direct limits (which he calls unions).

https://mathoverflow.net/questions/56363/list-of-classifying-spaces-and-covers

**Comment by prabau** [comment]
Definitely makes sense to have it as an alias then.
